### PR TITLE
hold notes custom alpha and other fixes

### PR DESCRIPTION
**Pull request by TheRealJake12**
yep thats right. I like the solid notes.

### CODE REVIEW

**Comment by glowsoony** [comment]
nice pull request

**Review by MemeHoovy — APPROVED** [review]

**Comment by TheRealJake12** [comment]
bolo accept the pr I fixed the options thing messing up when you press enter

File: source/Options.hx [hunk 1]
```diff
@@ -2321,6 +2321,56 @@ class ResetScoreOption extends Option
...
+	}
+
+	public override function press():Bool
+	{
+		return false;
+	}
+
+	private override function updateDisplay():String
+	{
+		return (FlxG.save.data.alpha ? "Scroll Alpha" : "Scroll Alpha");
```
Comment (Luisinhi010) [inline source/Options.hx:2340]: change this to 
return "Scroll Alpha" ;

dont need to update a text that is the same.

**Review by Luisinhi010 — APPROVED** [review]

File: source/Options.hx (hunk 1 above)
Comment (TheRealJake12) [reply source/Options.hx:2340]: 👍🏻

**Pull request closed without merging**